website/docs/docs/cloud/enable-dbt-copilot.md

```diff
@@ -32,4 +32,20 @@ dbt Copilot is only available at an account level after your organization has si
 
 Note: To disable (only after enabled), repeat steps 1 to 3, toggle off in step 4, and repeat step 5.
 
-<Lightbox src="/img/docs/deploy/example-account-settings.png" width="90%" title="Example of the 'Enable account access to AI-powered feature' option in Account settings" />
\ No newline at end of file
+<Lightbox src="/img/docs/deploy/example-account-settings.png" width="90%" title="Example of the 'Enable account access to AI-powered feature' option in Account settings" />
+
+### Bringing your own OpenAI API Key (BYOK)
+
+Once AI features have been enabled, you can provide your own organization's OpenAI API key. dbt Cloud will then your OpenAI account and terms to power dbt CoPilot. Note that this will incur billing charges to your organization from OpenAI for requests made by CoPilot.
```

Comment:
```suggestion
Once AI features have been enabled, you can provide your organization's OpenAI API key. dbt Cloud will then leverage your OpenAI account and terms to power dbt CoPilot. This will incur billing charges to your organization from OpenAI for requests made by dbt CoPilot.
```